src/tests/auth.test.ts

```diff
@@ -1,17 +1,10 @@
-import { Wrapper } from '../wrapper/index.js'
-import type { URLComponentsType } from '../types/general_types.js'
+import { getAPI } from './setupTests'
 
 describe('AuthTests', () => {
-    const urlComponents: URLComponentsType = {
-        host: 'localhost',
-        port: '9090',
-        version: '0.3.0',
-    }
-
-    const api = new Wrapper(urlComponents, 'rods', 'rods')
+    const api = getAPI()
```

Comment:
Consider renaming `getAPI` to something that better expresses the intent. For example:
```js
const api = createClientForTesting();
```